packages/composite-checkout-wpcom/src/components/wp-contact-form.js

```diff
@@ -14,11 +14,76 @@ import { useHasDomainsInCart } from '../hooks/has-domains';
 import Field from './field';
 import { SummaryLine, SummaryDetails, SummarySpacerLine } from './summary-details';
 import { LeftColumn, RightColumn } from './ie-fallback';
+import { prepareDomainContactDetails } from '../types';
 
-export default function WPContactForm( { summary, isComplete, isActive } ) {
-	const isDomainFieldsVisible = useHasDomainsInCart();
+const contactDetailsFormat = ( { hasDomainsInCart } ) => {
```

Comment:
We could move this to the bottom of the file by changing it to use the `function` keyword instead of `const`. That way the main export of this file still comes first.